test/e2e/traces_multi_pipeline_max_pipeline_test.go

```diff
@@ -75,24 +72,17 @@ var _ = Describe(suite.ID(), Label(suite.LabelTraces), Ordered, func() {
 
 				Expect(kitk8s.CreateObjects(ctx, k8sClient, pipeline.K8sObject())).Should(Succeed())
 
-				verifiers.TraceCollectorConfigShouldNotContainPipeline(ctx, k8sClient, pipelineName)
-
-				var fetched telemetryv1alpha1.TracePipeline
-				key := types.NamespacedName{Name: pipelineName}
-				Expect(k8sClient.Get(ctx, key, &fetched)).To(Succeed())
-
-				configurationGeneratedCond := meta.FindStatusCondition(fetched.Status.Conditions, conditions.TypeConfigurationGenerated)
-				Expect(configurationGeneratedCond).NotTo(BeNil())
-				Expect(configurationGeneratedCond.Status).Should(Equal(metav1.ConditionFalse))
-				Expect(configurationGeneratedCond.Reason).Should(Equal(conditions.ReasonMaxPipelinesExceeded))
-
-				pendingCond := meta.FindStatusCondition(fetched.Status.Conditions, conditions.TypePending)
-				Expect(pendingCond).NotTo(BeNil())
-				Expect(pendingCond.Status).Should(Equal(metav1.ConditionTrue))
-				Expect(pendingCond.Reason).Should(Equal(conditions.ReasonMaxPipelinesExceeded))
+				assert.TracePipelineHasCondition(ctx, k8sClient, pipelineName, metav1.Condition{
+					Type:   conditions.TypeConfigurationGenerated,
+					Status: metav1.ConditionFalse,
+					Reason: conditions.ReasonMaxPipelinesExceeded,
+				})
 
```

Comment:
we're not checking the absence of the running condition anymore.

Reply:
You are right, will add it